EIPS/eip-3157.md

```diff
@@ -0,0 +1,265 @@
+---
+eip: 3157
```

Comment:
I'm generally pretty against cherry picking EIP numbers.  The reason for this is that it is a slippery slope toward people doing things like spamming the repository to get the issue number they want (a problem we have had in the past) and it is easier to just say that you should always just use whatever number the editors assign (which is usually the PR number).

That being said, we have allowed some cherry picking in the past and other editors have expressed less concern than me.

Reply:
I would prefer the number be updated to 3234.

Reply:
Fixed